test/e2e/manifest-hybrid.yaml

```diff
@@ -0,0 +1,29 @@
+# TODO: make sure this is the ideal configuration
+StorageClass:
+  FromExistingClassName: thin-csi
+SnapshotClass:
+  FromName: true
+DriverInfo:
+  Name: csi.vsphere.vmware.com
+  SupportedFsType:
+    xfs: {}
+    ext4: {}
+  SupportedMountOption:
+    dirsync:
+  SupportedSizeRange:
+    Min: 1Gi
+    Max: 64Ti
+  Capabilities:
+    persistence: true
+    fsGroup: true
+    block: true
+    exec: true
+    volumeLimits: false
+    controllerExpansion: true
+    nodeExpansion: true
+    snapshotDataSource: false
+    multipods: false
+    multiplePVsSameID: false
+    readWriteOncePod: true
+NodeSelectors:
+  node.openshift.io/platform-type: "vsphere"
```

Comment:
This manifest is used and deseralized by csi e2e testsuite and afaik it doesn't support `NodeSelectors` fields - https://github.com/gnufied/kubernetes/blob/bump-recovery-feature-ga/test/e2e/storage/external/external.go#L51

Reply:
correct, i'm working on adding that support based on our meeting a few weeks ago: https://github.com/openshift/kubernetes/pull/2357